python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -125,6 +126,11 @@ def __init__(
         if self.name not in self._valid_ops:
             raise NotImplementedError(f"Temporal function {self.name}")
 
+        if self.name is TemporalFunction.Name.ToString and plc.traits.is_duration(
+            self.children[0].dtype
+        ):
+            raise NotImplementedError("ToString is not supported on duration types")
```

Comment:
Can we not use `plc.strings.convert.convert_durations.from_durations`?

Reply:
The issue is that the only two formats for supported by polars for `ToString` on duration types are `iso` and `polars`, both native formats that I don't think `from_durations` can readily replicate.

Reply:
Gotcha thanks!